src/components/DropDownButton/index.tsx

```diff
@@ -23,7 +23,7 @@ import { Tooltip } from '../Tooltip';
 
 type MenuItemType<T> = MenuItemWithTrackingProps &
 	T & {
-		tooltip?: string | Omit<TooltipProps, 'children'>;
+		tooltip?: string | Omit<TooltipProps, 'children'> | null;
```

Comment:
```suggestion
		tooltip?: Omit<TooltipProps, 'children'> | TooltipProps['title'];
```

Reply:
🤔 because TooltipProps['title'] does not have null:

`title?: string | undefined;`